widget/ui/src/components/NotFound/NotFound.tsx

```diff
@@ -9,9 +9,10 @@ import { Typography } from '../Typography';
 import { Container } from './NotFound.styles';
 
 export function NotFound(props: NotFoundPropTypes) {
+  const { hasIcon = true } = props;
   return (
     <Container>
-      <SearchIcon color="secondary" size={26} />
+      {hasIcon && <SearchIcon color="secondary" size={26} />}
```

Comment:
Maybe it is better to receive the icon as prop and show the default icon if it is not available.

Reply:
You're right, fixed